roles/satellite/tasks/sync_all_repos.yml

```diff
@@ -0,0 +1,6 @@
+---
+- include_tasks:
+    file: internals/facts-gathering.yml
```

Comment:
Are any facts defined in these files necessary for repository-sync ?

AFAIK `internals/repository-sync.yml ` only collects all the repositories defined in all contentviews in the inventory, and synchronizes them. There should be no fact involved in that.